build/escargot.cmake

```diff
@@ -31,6 +31,7 @@ ELSEIF (${ESCARGOT_OUTPUT} STREQUAL "shell_test")
     SET (ESCARGOT_CXXFLAGS ${ESCARGOT_CXXFLAGS} ${ESCARGOT_CXXFLAGS_SHELL} ${ESCARGOT_DEFINITIONS_TEST})
     SET (ESCARGOT_LDFLAGS ${ESCARGOT_LDFLAGS} ${ESCARGOT_LDFLAGS_SHELL})
     SET (ESCARGOT_DEFINITIONS ${ESCARGOT_DEFINITIONS} ${ESCARGOT_DEFINITIONS_SHELL})
+    add_definitions(-DESCARGOT_ENABLE_TEMPORAL)
```

Comment:
`Temporal` feature is enabled only for test mode.
Would you please enable it by default?
(this patch already has a lot of modifications, so fix it at the next patch if you want)